servers/visual/rasterizer.h

```diff
@@ -930,6 +930,7 @@ class Rasterizer {
 	virtual RasterizerScene *get_scene()=0;
 
 	virtual void initialize()=0;
+	virtual void update_system_fbo(uint32_t p_system_fbo)=0;
```

Comment:
lol... well I managed to get it 'working' on OSX, I'll see if I can get the audio stuff to compile on iPhone later tonight. Family time first:)